packages/gestalt/src/ButtonToggle.tsx

```diff
@@ -134,34 +163,115 @@ const ButtonToggleWithForwardRef = forwardRef<HTMLButtonElement, Props>(function
     [focusStyles.accessibilityOutline]: !disabled && isFocusVisible,
   });
 
+  // Consume GlobalEventsHandlerProvider
+  const { buttonToggleHandlers } = useGlobalEventsHandlerContext() ?? {
+    buttonToggleHandlers: undefined,
+  };
+
+  const bgColor: 'red' | 'transparent' = color instanceof Array ? 'red' : color;
```

Comment:
i dont understand this line. seems unnecessary.

Reply:
Fixed!